fdbclient/ReadYourWrites.actor.cpp

```diff
@@ -2103,6 +2135,8 @@ void ReadYourWritesTransaction::resetRyow() {
 	versionStampKeys = VectorRef<KeyRef>();
 	nativeReadRanges = Standalone<VectorRef<KeyRangeRef>>();
 	nativeWriteRanges = Standalone<VectorRef<KeyRangeRef>>();
+	specialKeySpaceWriteMap.rawErase(specialKeys); // TODO : check rawErase
```

Comment:
Can we add/use a `reset` method instead? I think that would be more idiomatic

Reply:
Added a TODO. Will add a `reset()` in the following pr.
Just feel this pr already contains many files. To make it easier for review, will make the change in the next pr.